src/Bridges/bridgeoptimizer.jl

```diff
@@ -209,19 +209,59 @@ function MOI.addconstraint!(b::AbstractBridgeOptimizer, f::MOI.AbstractFunction,
         MOI.addconstraint!(b.model, f, s)
     end
 end
-function MOI.canmodifyconstraint(b::AbstractBridgeOptimizer, ci::CI, change)
-    if isbridged(b, typeof(ci))
-       MOI.canmodifyconstraint(b.bridged, ci, change) && MOI.canmodifyconstraint(b, MOIB.bridge(b, ci), change)
+function MOI.canmodify(b::AbstractBridgeOptimizer, ::Type{C}, ::Type{Chg}) where {C<:CI,Chg<:MOI.AbstractFunctionModification}
+    if isbridged(b, C)
+       MOI.canmodify(b.bridged, C, Chg)
+       # TODO(@blegat): how should the following be implemented?
+       # && MOI.canmodify(b, MOIB.bridge(b, ci), Chg)
```

Comment:
Okay, @blegat new question. Is this necessary and in what use-case? It makes tests like
https://github.com/JuliaOpt/MathOptInterface.jl/pull/388/files#diff-5d9f4af23e8ec976db542b6b9af239c4R135
break, but everything else passes. There are also things like
https://github.com/JuliaOpt/MathOptInterface.jl/pull/388/files#diff-91d11cfb910f6ae0036ae89cac114d3eR144
Are they necessary?

Reply:
Yes, it checks that the bridge supports the modification. Currently, only the interval bridge supports modification

Reply:
I missed the second part to the question. If they are necessary, how do we implement this using only type information...

Reply:
You need `bridgetype` :)

Reply:
I have pushed a fix